mlflow/R/mlflow/Dockerfile.r-devel

```diff
@@ -1,4 +1,4 @@
-FROM rocker/r-ver:devel
+FROM r-base:latest
```

Comment:
Uses a debian base image that is compatible with the Docker installation on our CI systems